### ISSUE TITLE
Rollup is injecting JS not supported by IE11 (is IE11 supported?) 

### ISSUE THREAD

**Issue by GordonSmith** [rollup/rollup#3784]
- Rollup Version:  2.27.1 
- Operating System (or Browser):  IE11
- Node Version (if applicable):  NA
- Link to reproduction:  https://repl.it/@GordonSmith/QuaintPlumUtilities 

### Expected Behavior

RollupJS should not inject JS which is unsupported by IE11

### Actual Behavior

RollupJS is injecting `Object.assign` which is not supported in IE11

Generated code which includes Object.assign:
```
var _colorbrewer = /*#__PURE__*/Object.freeze(/*#__PURE__*/Object.assign(/*#__PURE__*/Object.create(null), colorbrewer$1, {
    'default': colorbrewer$1
}));
```

**Comment by GordonSmith** [comment]
FWIW, this is what tslib (typescript) uses as the polyfill (to solve the same issue) - 

```
var __assign = function() {
    __assign = Object.assign || function __assign(t) {
        for (var s, i = 1, n = arguments.length; i < n; i++) {
            s = arguments[i];
            for (var p in s) if (Object.prototype.hasOwnProperty.call(s, p)) t[p] = s[p];
        }
        return t;
    };
    return __assign.apply(this, arguments);
};
```

**Comment by shellscape** [comment]
I'd recommend injecting a polyfill into the bundle where needed, easily remedied from your end. 

IE11 is end of life next year and the list of sites no longer supporting it is growing exponentially. Best to prepare now for _no one_ supporting it in 11 months.

**Comment by GordonSmith** [comment]
@shellscape - while I agree with your sentiment re IE11, I am hitting this issue as both a consumer of bundles as well as a creator of bundles.

FWIW I have added pollyfills to code that I have control over and was simply reporting the issue here in an effort to help (I was a bit surprised to not see a similar report, or documentation saying IE11 no longer supported as of v2 etc.).

**Comment by lukastaegert** [comment]
This will finally be resolved in #4215
